tasks/acquia.xml

```diff
@@ -19,25 +19,33 @@
     <fail unless="build.dir" />
     <fail unless="build.env" />
 
-    <fail unless="acquia.accountname" />
```

Comment:
Because I've added the `acquia.xml` tasks to the `build.xml` template -- so including these lines at the top level of this file would mean that the `acquia.accountname` property would have to be configured in order to run any phing task.

Users would have to know to go and manually set this property in their config -- and not everybody will use these tasks anyway.

So, moving these checks back inside the specific targets made sense because this way, they only warn folks who run those targets without all of the required properties configured.

Reply:
Oooooh.  Good point!